pages/cockpit/installation-guide/installation-guide-introduction.adoc

```diff
@@ -12,3 +12,34 @@ The following sections explain how to prepare, install and configure your Cockpi
 
 NOTE: This Installation Guide provides information about how to install Gravitee.io components. For external components, such as
 MongoDB, see the relevant product documentation.
+
+Cockpit generates a certificate that will be used to authenticate installations (API/AM) that will connect. To create this certificate, you need to put private/public keys in keystore/truststore file and provide them using the following configuration attributes:
+
+[source,yaml]
+----
+keystore:
+  type: pkcs12
+  path: <path to the keystore file>
+  password: <password>
+truststore:
+  type: pkcs12
+  path: <path to the truststore file>
+  password: <password>
+----
+
+You also need a certificate to activate SSL for the WebSocket. Put this certificate in a `server.p12` Keystore and provide it through the following configuration attributes:
+
+[source,yaml]
+----
+ssl:
+  keystore:
+    type: pkcs12
+    path: <path to the keystore file>
+    password: <password>
+  truststore:
+    type: pkcs12
+    path: <path to the truststore file>
+    password: <password>
+----
+
+WARNING: Cockpit won't start if no keystore/truststore are provided.
```

Comment:
Do you think it could be a good idea to provide the steps to generate new certificates for dev purpose ?
I know that this is not supposed to be done this way for production, but maybe users discovering Cockpit might want to run it with a fake certificate / trustore before doing the real prod config :shrug:

Reply:
🤷  I've hesitated to do it. I don't know